code-challenges.test.js

```diff
@@ -25,27 +25,63 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
-// b) Create the function that makes the test pass.
+// describe(`divisibleByThree`, () => {
+//     it("decides if the number inside it is evenly divisible by three or not.", () => {
+// const object1 = { number: 15 }
+// // Expected output: "15 is divisible by three"
+// const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+// const object3 = { number: -7 }
+// // Expected output: "-7 is not divisible by three"
+//       expect(divisibleByThree(object1.number)).toEqual("is divisible by three");
+//       expect(divisibleByThree(object2.number)).toEqual("is divisible by three");
+//       expect(divisibleByThree(object3.number)).toEqual("is not divisible by three");
```

Comment:
The prompt says the function should take an object as an argument. Here, you are passing a number as an argument.